src/main/resources/lib/customizable-header/links.jelly

```diff
@@ -1,14 +1,43 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
   <j:if test="${it.hasLinks()}">
-    <template id="custom-header-icons">
-      <l:icon id="external-link" src="symbol-open-outline plugin-ionicons-api" class="icon-xs"/>
-    </template>
-    <div class="custom-header__app-nav">
-      <button class="custom-header__app-nav-button" data-href="${rootURL}/customizable-header/getLinks">
-        <l:icon src="symbol-apps-sharp plugin-ionicons-api" class="icon-md"/>
-      </button>
-    </div>
-    <script src="${resURL}/plugin/customizable-header/js/bundles/app-nav.js" type="text/javascript"></script>
+    <j:set var="tooltip">
+      <div class="jenkins-dropdown">
+        <j:forEach var="link" items="${it.links}">
+            <j:if test="${link.type == 'separator'}">
+                <j:choose>
+                    <j:when test="${link.title == null or link.title eq ''}">
+                        <div class="jenkins-dropdown__separator" />
+                    </j:when>
+                    <j:otherwise>
+                        <div class="jenkins-dropdown__heading">
+                            ${link.title}
+                        </div>
+                    </j:otherwise>
+                </j:choose>
+            </j:if>
+            <j:if test="${link.type == 'link'}">
+                <a href="${link.url}" class="jenkins-dropdown__item">
+                    <div class="jenkins-dropdown__item__icon">
+                        <j:out value="${link.iconXml}" escapeText="false" />
```

Comment:
iconXML might not be defined, e.g. when the chosen image is not a symbol but a simple png

Reply:
I reverted the change to make that a tooltip for now. The tooltip is placed a bit to the right which increases the chance that when moving the mouse the menu gets closed. Happened to me several times.